CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.23.1 FATAL_ERROR)
+
+enable_language(C CXX)
+
+include(fetch_rapids.cmake)
+
+include(rapids-cmake)
+include(rapids-cpm)
+include(rapids-find)
+
+rapids_cpm_init()
+
+rapids_find_package(
+  CUDAToolkit 12.2 REQUIRED
+)
+
+project(nvjitlink)
```

Comment:
Put the languages here. Should also specify a version.

Reply:
```suggestion
project(
  nvjitlink
  LANGUAGES C CXX
)
```